src/stirling/source_connectors/socket_tracer/protocols/BUILD.bazel

```diff
@@ -45,5 +45,6 @@ pl_cc_library(
         "//src/stirling/source_connectors/socket_tracer/protocols/nats:cc_library",
         "//src/stirling/source_connectors/socket_tracer/protocols/pgsql:cc_library",
         "//src/stirling/source_connectors/socket_tracer/protocols/redis:cc_library",
+        "//src/stirling/source_connectors/socket_tracer/protocols/mqtt:cc_library",
```

Comment:
This should be saved for a later change. Most likely once we introduce the "mqtt trace bpf test"